spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/ImportExportControllerHelperService.java

```diff
@@ -570,7 +572,7 @@ public JSONObject librarySelectionTypesString(HttpSession session, JSONObject js
 
   public JSONObject changePlatformName(HttpSession session, JSONObject json) {
     try {
-      if (json.has("platform") && !json.get("platform").equals("")) {
+      if (json.has("platform") && !isStringEmptyOrNull((String) json.get("platform"))) {
```

Comment:
I strongly disagree. How do you convert from Object to String? There are two ways I see:
- just do instanceof String, in which case calling it on a StringBuffer or the like will return false
- call toString on it, in which case nearly every object will return true.